src/app/copy/block-copy/block-copy.component.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Component,
+  EventEmitter,
+  Input,
+  ViewEncapsulation
+} from '@angular/core';
+import { uniqueId } from 'lodash';
+
+import { CopyBase } from '../copy-base';
+
+import { CopyService } from '../copy-service/copy.service';
+
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'pfng-block-copy',
+  templateUrl: './block-copy.component.html',
+  styleUrls: ['./block-copy.component.less']
+})
+
+export class BlockCopyComponent extends CopyBase {
+  @Input('label') label: string;
+  @Input('expandBtnAriaLabel') expandBtnAriaLabel: string;
+  @Input('buttonLabel') buttonLabel: string = 'Copy';
+  @Input('expanded') expanded: boolean = false;
+
+  public uniqueID: string = uniqueId('pfng-block-copy-button-');
```

Comment:
I'd like to make this equal to `uniqueId('pfng-block-copy');`. That way, lodash's unique ID is not appended to the end of the string.

See below...